src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private final int height;
+    private final int firstSide;
+    private final int secondSide;
+
+    public IsoscelesTrapezoid(Color color, int height, int sideA, int sideB) {
+        super(color);
+        this.height = height;
+        this.firstSide = sideA;
+        this.secondSide = sideB;
+    }
+
+    @Override
+    public double calculateArea() {
+        return ((firstSide + secondSide) / 2.0) * height;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: Isosceles Trapezoid"
+                + ", area: "
+                + calculateArea() + " sq.units, firstSide: "
+                + firstSide + " units, secondSide: "
+                + secondSide + " units, height: "
+                + height + " units, color: "
+                + getColor().toString().toLowerCase());
```

Comment:
[you need to use method `name()` to get a string representation of the enum. ](https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced#use-name-for-getting-string-representation-of-enum-constants)